static/js/components/ManageDataForm.jsx

```diff
@@ -450,6 +477,13 @@ const ManageDataForm = ({ route }) => {
                 <SpectrumRow rowData={rowData} route={route} />
               ),
               expandableRowsOnClick: false,
+              // rowsExpanded: openedSpecRows,
+              // onRowExpansionChange: (currentRowsExpanded) => {
+              // console.log(currentRowsExpanded);
+              // console.log(allRowsExpanded);
+              // console.log(rowsExpanded);
+              // setOpenedSpecRows(currentRowsExpanded);
+              // },
```

Comment:
@kmshin1397 This is where I figured there should be some logic going into `onRowExpansionChange` but I couldn't figure out what exactly to do with it.